static/src/stylesheets/module/site-messages/_subscription-medium-banner.scss

```diff
@@ -390,3 +390,164 @@
 }
 
 /*********************** footer hacks END *********************/
+
+/*********************** gw Banner *********************/
+.site-message--gw-banner {
+    &.site-message--subscription-banner {
+        background-color: #3e474a;
+        padding: 30px 20px;
+    }
+
+    .site-message--subscription-banner__description {
+
+        @include mq($from: tablet) {
+            width: 55%;
+        }
+
+        @media (min-width: 781px) {
+            width: 60%;
+        }
+
+        @include mq($from: desktop) {
+            width: 50%;
+        }
+
+        @include mq($from: leftCol) {
+            width: 46%;
+        }
+
+        @include mq($from: wide) {
+            width: 47%;
+        }
+
+        p {
+            line-height: 1.4;
+        }
+    }
+
+    .site-message--subscription-banner__title,
+    .site-message--subscription-banner__description,
+    .site-message--subscription-banner__cta-dismiss,
+    .site-message--subscription-banner__sign-in,
+    .site-message--subscription-banner__sign-in-link,
+    .site-message--subscription-banner__subscriber-link,
+    .site-message--subscription-banner__close-button {
+        color: $brightness-97;
+    }
+
+    .site-message--subscription-banner__title {
+        padding-right: 44px;
+
+        @include mq($from: tablet) {
+            padding-right: 0px;
+        }
+    }
+
+    .site-message--subscription-banner__cta {
+        background: $highlight-main;
+        color: #121212;
+        width: 55%;
+    }
+
+    .site-message--subscription-banner__close-button svg {
+        border: 1px solid $brightness-97;
+        padding: 13px;
+        margin-top: -9px;
+    }
+
+    .site-message--subscription-banner__close-button svg path {
+        fill: $brightness-97;
+    }
+
+    .site-message--subscription-banner__gu-logo svg path {
+        fill: $brightness-97;
+    }
+
+    .site-message--packshot-container {
+        bottom: -30px;
+        margin: 10px 0 -30px;
+
+        @include mq($from: tablet) {
+            width: 51%;
+        }
+
+        @media (min-width: 1140px) {
+            width: 57%;
+        }
+
+        img {
+            max-width: 320px;
+
+            @include mq($from: tablet) {
+                max-width: 325px;
+                margin-right: 0;
+                margin-left: auto;
+            }
+
+            @media (min-width: 781px) {
+                max-width: 320px;
+            }
+
+            @media (min-width: 889px) {
```

Comment:
Why do we have these custom breakpoints? They concern me a little bit because we really ought to be trying to move to a more consistent responsive design that can handle different packshots and copy - these make it seem a bit more fragile to change.